internal/git/gitsync_processor_test.go

```diff
@@ -685,9 +687,20 @@ func Test_watchRepo(t *testing.T) {
 				RepoUrl:        "https://github.com/numaproj-labs/numaplane.git",
 				Path:           "invalid_path",
 				TargetRevision: "main",
+				Name:           "control-manifest",
 			}),
 			hasErr: true,
 		},
+		{
+			name: "Apply manifest from kustomize enabled repo",
+			gitSync: newGitSync(v1alpha1.RepositoryPath{
+				RepoUrl:        "https://github.com/numaproj/numaflow.git",
+				Path:           "config/namespace-install",
+				TargetRevision: "main",
+				Name:           "numaflow",
+			}),
+			hasErr: false,
+		},
```

Comment:
Maybe in https://github.com/numaproj-labs/numaplane-control-manifests.git or another repo that we control we could have all of the manifests we depend on for testing, rather than being dependent on this repository. I'll write an issue for that.

Reply:
https://github.com/numaproj-labs/numaplane/issues/101